packages/lisk-transactions/src/4_multisignature_transaction.ts

```diff
@@ -331,4 +332,65 @@ export class MultisignatureTransaction extends BaseTransaction {
 
 		return [];
 	}
+
+	public addMultisignature(
+		store: StateStore,
+		signatureObject: SignatureObject,
+	): TransactionResponse {
+		// Get the account
+		const account = store.account.get(this.senderId);
+		// Check if signature is not already there
+		if (this.signatures.includes(signatureObject.signature)) {
```

Comment:
this check is done in `validateMultisignatures`

Reply:
Removed.